src/skydb.ts

```diff
@@ -25,7 +25,7 @@ export async function getJSON(
     url: this.getSkylinkUrl(skylink),
   });
 
-  return { data: response.data, revision: entry.entry.revision };
+  return { json: response.data, revision: entry.entry.revision };
```

Comment:
It should be `data` so once we have `getText` it will have the same interface instead of `entry.text`. Why did you change it in the first place?